src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/barnsalder/OpprettOppgaverForBarnService.kt

```diff
@@ -83,4 +93,24 @@ class OpprettOppgaverForBarnService(
     private fun enhetForInnhentDokumentasjon(personIdent: String): Enhet {
         return familieIntegrasjonerClient.hentBehandlendeEnhetForBehandlingMedRelasjoner(personIdent).first()
     }
+
+    fun leggOppgaveIMappe(oppgaveId: Long) {
+        val oppgave = oppgaveClient.finnOppgaveMedId(oppgaveId)
+        if (oppgave.tildeltEnhetsnr == EF_ENHETNUMMER) { // Skjermede personer skal ikke puttes i mappe
+            val finnMappeRequest = FinnMappeRequest(
+                listOf(),
+                oppgave.tildeltEnhetsnr!!,
+                null,
+                1000
+            )
+            val mapperResponse = oppgaveClient.finnMapper(finnMappeRequest)
+            val mappe = mapperResponse.mapper.find {
+                it.navn.contains("EF Sak", true) &&
+                    it.navn.contains("Hendelser") &&
+                    it.navn.contains("62")
+            }
+                ?: error("Fant ikke mappe for uplassert oppgave (EF Sak og 01)")
```

Comment:
Kanskje en annen feilmelding? "Fant ikke mappe for hendelser" ?

Reply:
Det er mer forståelig. Rettet.